src/syrupy/report.py

```diff
@@ -8,18 +8,22 @@
     Dict,
     Iterator,
     List,
+    Optional,
+    Set,
+    Tuple,
 )
 
 import attr
-import pytest
+from _pytest.mark.expression import Expression
```

Comment:
Alternative to importing the protected class is to roll our own parsing of `-k` expressions.

Reply:
Just confirming this works in the min. version of pytest we support?

Reply:
Haven't will test that out, probably should have a way to automate the CI range test

Reply:
Yeah no this does not work with pytest 5...

Reply:
Changed this to include a naive version of the `Expression` class for compatibility with older pytest versions while still giving us some of the scaffolding for keyword snapshot inclusion